model/src/pyrenew/latent/infectionswithfeedback.py

```diff
@@ -81,7 +81,7 @@ def __init__(
             Infection feedback strength.
         infection_feedback_pmf : RandomVariable
             Infection feedback pmf.
-        infections_mean_varname : str, optional
+        infections_varname : str, optional
             Name to be assigned to the deterministic variable in the model.
             Defaults to "latent_infections".
```

Comment:
```suggestion
        infections_varname : str
            Name to be assigned to the deterministic variable in the model.
```

As per [DHM](https://github.com/CDCgov/multisignal-epi-inference/pull/205#pullrequestreview-2131069672)'s comment.